packages/unit-testing/README.md

```diff
@@ -5,13 +5,15 @@ Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestj
 The following software is required to use SuiteCloud Unit Testing package:
 - Node.js version 12.14.0 LTS or greater
 - [Jest](https://jestjs.io/docs/en/getting-started.html)
-- A `__tests__` folder in the root of your SDF project.
-- A `jest.config.js` file in the root of your SDF project.
+
+Also, to properly structure your test files, create the following elements in the root of your SDF project:
```

Comment:
Great! I would also change 'to properly structure your test files' to smth. like 'to properly structure and run your tests'